pkg/pool-weighted/test/foundry/GradualValueChange.t.sol

```diff
@@ -95,6 +108,19 @@ contract GradualValueChangeTest is Test {
         assertEq(mock.calculateValueChangeProgress(startTime, endTime), FP_ONE, "Should be complete after end time");
     }
 
+    function testCalculateValueChangeProgressSameStartEndTime() public {
+        uint256 startTime = block.timestamp + 100;
+        uint256 endTime = startTime;
+
+        assertEq(mock.calculateValueChangeProgress(startTime, endTime), 0, "Wrong initial progress");
```

Comment:
```suggestion
        // block.timestamp < start time. Should report 0% before the update begins.
        assertEq(mock.calculateValueChangeProgress(startTime, endTime), 0, "Wrong initial progress");
```
Optional extra comments to show that both branches are being tested. (One twice.)